plugins/build.gradle.kts

```diff
@@ -5,3 +5,8 @@ plugins {
 repositories {
     gradlePluginPortal()
 }
+
+java {
```

Comment:
Is this really needed?

Reply:
Well, I get like mismatched between Java versions and Kotlin options warning.

Reply:
OK!